server/models/video/sql/video/videos-id-list-query-builder.ts

```diff
@@ -598,6 +604,18 @@ export class VideosIdListQueryBuilder extends AbstractRunQuery {
     this.replacements.durationMax = durationMax
   }
 
+  private whereExcludeAlreadyWatched (userId: number) {
+    this.and.push(
+      'NOT EXISTS (' +
+      '  SELECT *' +
```

Comment:
`SELECT 1` for consistency (not sure if it's more efficient in this context, but we always prefer to not select columns when we don't need to so PostgreSQL doesn't have to fetch more data from the disk)